ocean_lib/data_provider/data_service_provider.py

```diff
@@ -105,6 +106,44 @@ def encrypt_files_dict(
 
         return ""
 
+    @staticmethod
+    @enforce_types
+    def encrypt_urls(files: list, encrypt_endpoint: str) -> Union[bytes, str]:
+        # FIXME: uncomment when Provider have support for this storage type
+        # data = str(list(map(lambda file: file.from_dict(), files)))
+
+        payload = json.dumps(
+            {
+                "document": json.dumps(
+                    [file.url for file in files], separators=(",", ":")
+                )
+            }
+        )
+        response = DataServiceProvider._http_method(
+            "post",
+            encrypt_endpoint,
+            data=payload,
+            headers={"content-type": "application/octet-stream"},
+        )
+
+        if not response or not hasattr(response, "status_code"):
+            return ""
```

Comment:
I think this function should throw an exception in this case.

Reply:
I agree